lib/core-server/src/cli/dev.ts

```diff
@@ -29,6 +29,7 @@ export async function getDevCli(packageJson: {
     .option('--ssl-key <key>', 'Provide an SSL key. (Required with --https)')
     .option('--smoke-test', 'Exit after successful start')
     .option('--ci', "CI mode (skip interactive prompts, don't open browser)")
+    .option('--no-open', 'Do not open Storybook automatically in the browser')
```

Comment:
Should this be `--open`? @yannbf

Reply:
According to [commander's docs](https://github.com/tj/commander.js/#other-option-types-negatable-boolean-and-booleanvalue) that's how you set a flag that is true by default and will be set to false when used. With this option set, `--open` will not be recognized, only `--no-open`. However, when would anyone ever use the flag `--open` if that is true by default? 🤔

Reply:
🚀